util/assets/a_assets-packr.go

```diff
@@ -1,14 +0,0 @@
-// Code generated by github.com/gobuffalo/packr. DO NOT EDIT.
```

Comment:
what's the deal with this file disappearing and reappearing all the time?

Reply:
I think it should not neven been committed. Probably files were not deleted due to interrupted codegen process